src/bidiMapper/domains/network/NetworkRequest.ts

```diff
@@ -361,8 +361,8 @@ export class NetworkRequest {
   /** Returns the HTTP status code associated with this request if any. */
   get statusCode(): number {
     return (
-      this.#response.extraInfo?.statusCode ??
       this.#response.info?.response.status ??
+      this.#response.extraInfo?.statusCode ??
```

Comment:
This is wrong... The extra info has the correct one if we return for example `304` not modified response.
Now it return 200 and brakes Puppeteer test. 
We need to find a way to compute which is correct.

Reply:
https://github.com/puppeteer/puppeteer/actions/runs/6706355305/job/18222699456?pr=11284

Reply:
For network interception, this is correct.

We'll need to add some extra conditions in that method then. If request isBlocked, keep it like this. Otherwise do it the other way around.